### PR TITLE
Fix more Rubocop warnings.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

CC @reitermarkus

### CODE REVIEW

File: Formula/gitlab-ci-multi-runner.rb [hunk 1]
```diff
@@ -43,8 +43,7 @@ def install
 
     cd dir do
       resource("prebuilt-x86_64.tar.xz").stage do
-        system *%W[go-bindata -pkg docker -nocompress -nomemcopy -nometadata
-                   -o #{dir}/executors/docker/bindata.go prebuilt-x86_64.tar.xz]
+        system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy", "-nometadata", "-o", "#{dir}/executors/docker/bindata.go", "prebuilt-x86_64.tar.xz"
```
Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: I feel like crying.

Comment (MikeMcQuaid) [inline Formula/gitlab-ci-multi-runner.rb]: Personally I think this is way nicer to non-Rubyists.

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: can we at least wrap these to keep from going passed the eightieth column ...

Comment (MikeMcQuaid) [inline Formula/gitlab-ci-multi-runner.rb]: 👍

File: Formula/gitlab-ci-multi-runner.rb [hunk 2]
```diff
@@ -43,8 +43,10 @@ def install
 
     cd dir do
       resource("prebuilt-x86_64.tar.xz").stage do
-        system *%W[go-bindata -pkg docker -nocompress -nomemcopy -nometadata
-                   -o #{dir}/executors/docker/bindata.go prebuilt-x86_64.tar.xz]
+        system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy",
+                             "-nometadata", "-o",
```
Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: @DomT4 are these going to induce :cold_sweat: if the alignment is not under the first argument?

**Review by DomT4 — APPROVED** [review]

File: Formula/gitlab-ci-multi-runner.rb (hunk 2 above)
Comment (DomT4) [inline Formula/gitlab-ci-multi-runner.rb]: No, in this case I think it's fine, given `go-bindata` is the executable and `-pkg` is the first "true" argument.

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: a new rule cometh! hark the :angel:s sing!

File: Formula/pypy.rb [hunk 3]
```diff
@@ -64,11 +64,13 @@ def install
       package_args = %w[--archive-name pypy --targetdir . --nostrip]
       package_args << "--without-gdbm" if build.without? "gdbm"
       system python, "package.py", *package_args
-      system(*%W[tar -C #{libexec} --strip-components 1 -xzf pypy.tar.bz2])
+      system("tar", "-C", libexec.to_s, "--strip-components", "1", "-xzf", "pypy.tar.bz2")
```
Comment (ilovezfs) [inline Formula/pypy.rb]: probably want to drop the parentheses here, too, I imagine

Comment (MikeMcQuaid) [inline Formula/pypy.rb]: Done.

**Pull request closed without merging**